rocca/envs/gym_chase/chase_env.py

```diff
@@ -54,17 +54,13 @@ class ChaseEnv(gym.Env):
     receives a reward of 1.
     """
 
-    metadata = {
-        'render.modes': ['human', 'rgb_array'],
-        'video.frames_per_second': FPS
-    }
+    metadata = {"render.modes": ["human", "rgb_array"], "video.frames_per_second": FPS}
 
     def __init__(self):
         self.action_space = spaces.Discrete(4)
-        self.observation_space = spaces.Tuple((
-            spaces.Discrete(2),  # Agent Position
-            spaces.Discrete(3)  # Food pellet
-        ))
+        self.observation_space = spaces.Tuple(
+            (spaces.Discrete(2), spaces.Discrete(3))  # Agent Position  # Food pellet
```

Comment:
That auto-formatting was a bit dumb.

Reply:
Just in case, there is an option of turning formatting off locally by adding `# fmt: off` above that expression - of course it shouldn’t be used liberally.

Reply:
In general, `black` will base its changes on whether some line is long enough to be broken down into segments - this particular one is not long enough. It’s unfortunate that comments got garbled but this can be fixed easily by moving the comment up.

Reply:
Even garbled it's sufficiently obvious that the comments mean, so I decided to let it.  Good to know about `# fmt: off` though.